nid_validator/usa_validator/tests/test_validator.py

```diff
@@ -1,21 +1,28 @@
 from django.test import TestCase
 from usa_validator.validator import USANIDValidator
+import factory
+
+class USANIDValidatorFactory(factory.Factory):
+    class Meta:
+        model = USANIDValidator
+
+    nid = factory.Faker('random_number', digits=9, fix_len=True)
```

Comment:
Sweep: Add a new file in the tests directory called `factories.py`, including all the factory classes to it.